app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,38 @@
+
+<h1>Public Recipes</h1>
+
+<div id="recipes">
+  <% @recipes.each do |recipe| %>
+    <% @precipe = RecipeFood.where(recipe_id: recipe.id)%>
+    <div id="<%= dom_id recipe %>">
+  
+        <div class="recipe-content">
+           <div class="recipe-c-l">
+              <h3><%= recipe.name%></h3>          
+              <p>
+                <strong>by:</strong>
+                <%= recipe.user.name %>
+              </p>
+           </div>
+           <div class="recipe-c-r">
+            <p>
+      <strong>Total food items:</strong>
+      <%=@precipe.length%>
+      </p>
+       <p>
+        <strong>Total price:</strong>
+        <% @totalv = 0 %>
+        <%@precipe.each do |pr|%>
+        <% @totalv += pr.food.price.to_i * pr.quantity.to_i %>
+        <%end%>   
+        $<%=@totalv%>
+      </p>
+      <%= link_to "View the recipe", recipe %>
```

Comment:
[REQUIRED] KINDLY ENSURE THAT YOUR DESIGN MATCHES THE WIREFRAME BY FOLLOWING THE SUGGESTION IN THE IMAGE BELOW
![image](https://github.com/OnlyUnknown/Recipe-app-ROR/assets/62938735/9decd843-85f8-4a7e-9f32-15620f365205)